rdkit/VLib/Filter.py

```diff
@@ -3,10 +3,10 @@
 #  Copyright (C) 2003 Rational Discovery LLC
 #     All Rights Reserved
 #
-import sys
+
+import six
```

Comment:
Yes, this is the important point for me: why add the external dependency when we don't need it.